src/util/FetchUtils.js

```diff
@@ -46,6 +46,8 @@ export default class FetchUtils {
                 // that it's a redirect to a login page. If so, then we then reload the whole
                 // page to let the user log in again.
                 window.location.reload();
+              } else if (text.startsWith('<html') && text.includes('j_security_check')) {
```

Comment:
Isn't it possible for there to be a doctype at the start of the text? Would it make sense to check for the presence of '<html' instead of checking for it at the beginning?

Reply:
Perhaps, I copied that conditional from Search Analytics. I could probably just do the `text.includes('j_security_check')`.